bin/safe_dir_rotation

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
```

Comment:
Maybe add a usage statement or a couple of comments on how this should be invoked?